save-backend/src/main/kotlin/com/saveourtool/save/backend/security/TestSuitePermissionEvaluator.kt

```diff
@@ -0,0 +1,25 @@
+package com.saveourtool.save.backend.security
+
+import com.saveourtool.save.entities.TestSuite
+import org.springframework.security.core.Authentication
+import org.springframework.stereotype.Component
+
+/**
+ * Class that is capable of assessing user's permissions regarding test suites.
+ */
+@Component
+class TestSuitePermissionEvaluator {
+    /**
+     * fixme: just a remark for future work
+     *
+     * @param testSuite test suite that is being requested
+     * @param authentication
+     * @return true if user with [authentication] can access [testSuite], false otherwise
+     */
+    fun canAccessTestSuite(
+        testSuite: TestSuite,
+        authentication: Authentication?
+    ) = authentication?.let {
+        true
+    } ?: false
```

Comment:
I think `authentication == null` looks much more clear. Does diktat force `let` here?

Reply:
oh, don't think diktat forces us to do it. The point is that it is planned to put something more suitable instead of `true`